### ISSUE TITLE
SMA doesn't handle errors when using operation endpoint

### ISSUE THREAD

**Issue by anonymouse64** [edgexfoundry/edgex-go#769]
For example, when using SMA with docker, attempting to stop a non-existent service returns HTTP 200 and this message:

```
Done. Stopped the requested services.
```

To reproduce do:
```
curl --header "Content-Type: application/json" --request POST --data '{"action":"stop","services":["edgex-not-a-thing"]}' localhost:48090/api/v1/operation; echo
```

**Comment by tsconn23** [comment]
@akramtexas Confirmed this is still an issue. Is there a plan for resolution?

**Comment by akramtexas** [comment]
> @akramtexas Confirmed this is still an issue. Is there a plan for resolution?

@tsconn23 Yep, this is right next up on my `TO-DO` list, and I plan on resolving this as soon as I'm done with the work underway for `integrating sys-mgmt with device-sdk-go` 👍

**Comment by akramtexas** [comment]
This work is next up as I wrap up the `sys-mgmt` integration work with `device-sdk-go`.

**Comment by akramtexas** [comment]
Thanks for picking up the bug-fix @brandonforster 👍

**Comment by brandonforster** [comment]
Closed by #1184
